packages/jest-phabricator/src/index.js

```diff
@@ -53,9 +53,27 @@ module.exports = function(results: AggregatedResult): PhabricatorReport {
   const formatter = (coverage, reporter) => coverageMap;
   const report = formatTestResults(results, formatter);
 
-  return Object.assign((Object.create(null): any), results, {
+  const returnPhabricatorReport: PhabricatorReport = {
```

Comment:
Could you instead of creating a new variable simply return the object here? I think you don't need to type-cast it at all in this case :)